app/boards/arm/bdn9/bdn9_rev2.dts

```diff
@@ -118,7 +127,7 @@
 &flash0 {
 	/*
 	 * For more information, see:
-	 * http://docs.zephyrproject.org/latest/guides/dts/index.html#flash-partitions
+	 * http: //docs.zephyrproject.org/latest/guides/dts/index.html#flash-partitions
```

Comment:
Errant space -- I am guessing this is `clang-format`'s doing?

Reply:
Some formatter fubar'ing things. Fixed. Thanks!